django_ipam/base/admin.py

```diff
@@ -1,22 +1,30 @@
+import csv
 from ipaddress import IPv4Network, IPv6Network, ip_address
 
 import swapper
 from django import forms
 from django.contrib.admin import ModelAdmin
 from django.http import HttpResponse
-from django.urls import reverse
+from django.shortcuts import render
+from django.urls import path, re_path, reverse
 from django.utils.translation import gettext_lazy as _
 from openwisp_utils.admin import TimeReadonlyAdminMixin
 
+from .forms import IpAddressImportForm
+
 Subnet = swapper.load_model("django_ipam", "Subnet")
 IpAddress = swapper.load_model("django_ipam", "IpAddress")
 
 
 class AbstractSubnetAdmin(TimeReadonlyAdminMixin, ModelAdmin):
     change_form_template = "admin/django-ipam/subnet/change_form.html"
+    change_list_template = "admin/django-ipam/subnet/change_list.html"
+    app_name = "django_ipam"
 
     def change_view(self, request, object_id, form_url="", extra_context=None):
         instance = Subnet.objects.get(pk=object_id)
+        ipaddress_add_url = 'admin:{0}_ipaddress_add'.format(self.app_name)
+        ipaddress_change_url = 'admin:{0}_ipaddress_change'.format(self.app_name)
```

Comment:
is this for fixing the sample app? If yes move this change into a different patch from a different branch